e2e/fixtures/ssg-performance/src/pages/[slug].tsx

```diff
@@ -1,8 +1,9 @@
+import type { PageProps } from 'waku/router';
 import { Path } from '../Path.js';
 
-export default async function Test() {
+export default async function Test({ path }: PageProps<'/[slug]'>) {
   await new Promise((resolve) => setTimeout(resolve, 1000));
-  return <Path />;
+  return <Path path={path} />;
```

Comment:
@dai-shi are you ok with this change to the test? I think for testing ssg-performance it still fits that description, but it is different, so double checking seems good.

Reply:
Yeah, this is unavoidable because of #1034 change.
It should still be fine for `ssg-performance` spec.
context test should be somewhere else. Besides, using `cache` for server context isn't recommended.